pkg/util/mount/mount_windows.go

```diff
@@ -0,0 +1,131 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+type Mounter struct {
+	mounterPath string
+}
+
+func (mounter *Mounter) Mount(source string, target string, fstype string, options []string) error {
+	if source == "tmpfs" || len(target) < 3 {
+		glog.Infof("azureMount: Skip mounting source (%q), target (%q)\n, with arguments (%q) in windows", source, target, options)
+		return nil
+	}
+
+	if !strings.HasPrefix(target, "c:") && !strings.HasPrefix(target, "C:") {
+		target = "c:" + target
```

Comment:
It's already normaliazed in k8s common code, like below:
("\\var\\lib\\kubelet\\pods\\f6590c44-7e6b-11e7-8f44-000d3a31dac4\\volumes\\kubernetes.io~secret\\default-token-rrrc5") 
while it's missing drive letter in the beginning.